mogle1-milwaukee.md

```diff
@@ -0,0 +1,8 @@
+# Milwaukee, WI
+
+### Restaurants
+
```

Comment:
What about Culvers :hamburger: , they're from Wisconsin right? :cheese:

Reply:
@jrdisc2 added some restaurants, let me know if this is good to :ship:

Reply:
You bet! :cool: